firmware/release-notes.md

```diff
@@ -70,32 +70,32 @@ https://www.raspberrypi.org/documentation/hardware/raspberrypi/bcm2711_bootloade
 
 ## 2020-09-03 Only use green LED for error status in bootloader - BETA
    * Turn the green LED on and leave it on unless an error code occurs.
-     Previously, SD activity was displayed but that plus muxing with the
+     Previously, SD activity was displayed but that plus mixing with the
```

Comment:
@timg236 I think this should have been left as `muxing` ?

Reply:
Whoops! I should've told Word to ignore those. Welp